.changeset/giant-timers-ring.md

```diff
@@ -0,0 +1,5 @@
+---
+'@primer/components': minor
+---
+
+Adds a RadioButton component
```

Comment:
Curious if we should call this `Radio` instead of `RadioButton`? Don't have strong feelings either way, just wanted to raise the question in case there is a "correct" answer.

Reply:
We call the checkbox input a `Checkbox`, so I'm with Cole on this one.

Reply:
I called this `Radio` initially  and later renamed to `RadioButton` 😅 , as that appeared to be the common naming convention used by many other component libraries for this type of input. There's [a bit history around the 'button' aspect to it too](https://www.nngroup.com/articles/radio-buttons-default-selection/#articleBody). Shall I change it to Radio? Or it may be worth checking in with wider group too, as we'd need to coordinate the same naming with PVC and CSS eventually too.

Reply:
Update: Changed to `Radio` now.